lib/client.js

```diff
@@ -97,7 +97,7 @@ class RPClient {
                 content: fileData
             };
             rest.post(url, {
-                data: _self.buildMultiPartStream(saveLogRQ, file, boundary),
+                data: _self.buildMultiPartStream([saveLogRQ], file, boundary),
```

Comment:
I believe, we should fix agents before merging it. Seems like they already put array here